src/app/workout/tabs.tsx

```
@@ -0,0 +1,67 @@
+'use client'
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Box from '@mui/material/Box';
+import Workouts from "@/app/workout/workouts/workouts";
+import Workout from "@/types/Workout";
```

ESLint

[error] 'Workout' is defined but never used.

(@typescript-eslint/no-unused-vars)